examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -43,3 +46,19 @@
 
 # Now let's take a look at the spatial distributions of the psd.
 epochs.plot_psd_topomap(ch_type='grad', normalize=True)
+
+# Alternatively, you may also create PSDs from Epochs objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(epochs[:5], low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
But this is an example, not a test. We have to be careful with examples because users might see whatever is done as "the right thing to do". And users probably don't usually want to use only the first 5 epochs. If you want, you can add a comment above this line saying that only 5 epochs are used for speed if necessary.

How bad is the slowdown for using all epochs?